.github/workflows/e2e-main.yaml

```diff
@@ -97,33 +104,20 @@ jobs:
       - name: Build Podman Desktop for E2E tests
         working-directory: ./podman-desktop
         run: |
-          yarn --frozen-lockfile
-          yarn test:e2e:build
-
-      - name: Get yarn cache directory path
-        working-directory: ${{ env.REPOSITORY }}
-        id: yarn-cache-dir-path
-        run: echo "dir=$(yarn cache dir)" >> ${GITHUB_OUTPUT}
-
-      - uses: actions/cache@v4
-        id: yarn-cache
-        with:
-          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
-          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-yarn-
+          pnpm install
+          pnpm test:e2e:build
 
-      - name: Execute yarn in Red Hat Account Extension
+      - name: Execute pnpm in Red Hat Account Extension
         working-directory: ${{ env.REPOSITORY }}
         run: |
-          yarn add -D @podman-desktop/tests-playwright@next
-          yarn --frozen-lockfile
+          pnpm add -D @podman-desktop/tests-playwright@1.13.0-202408291153-ccbdd9b
```

Comment:
Please revert this line, we need to have @next as it points to the latest released npm package version.

Reply:
Can we keep it for now? I'll update it as part of migrating to use PW runner.

Reply:
Ah, I missed your comment. ok. god mode then! :)